static/js/translation.js

```diff
@@ -0,0 +1,69 @@
+
+//load initial language file.
+languageStartup();
+
+function languageStartup(){
+    var url = window.location.href;
+    var langChoice = url.split('#lang=')[1] || "en";
```

Comment:
I would recommend using `window.location.hash` instead.
As a side note, just wondering if there was a specific reason to use URL fragments instead of query parameters?

Reply:
Noted, will change, and in regards to the URL fragment, there isn't a specific reason for using fragments in comparison to query string params, is there much of a difference in our case?

Reply:
Well, once difference is that fragments are "invisible" to the server (the server doesn't see any part after the `#` in the request), and they are processed in the browser. So I didn't know if that could be part of the reason.

Apart from that, in most cases "settings" processed by the page are usually added as query parameters, but I don't think it actually makes much of a difference.

Reply:
Changed to use query parameters instead since that makes more sense.